shared/metrics/src/main/java/io/pravega/shared/metrics/DynamicLoggerImpl.java

```diff
@@ -41,7 +41,7 @@ public DynamicLoggerImpl(MetricsConfig metricsConfig, MetricRegistry metrics, St
         this.metrics = metrics;
         this.underlying = statsLogger;
         this.cacheSize = metricsConfig.getDynamicCacheSize();
-        this.cacheEvictionDuration = metricsConfig.getDynamicCacheEvictionDurationMs();
+        this.cacheEvictionDuration = metricsConfig.getDynamicCacheEvictionDurationMinutes();
 
         countersCache = CacheBuilder.newBuilder().
```

Comment:
You take a value that is `minutes` and declare that it is `TimeUnit.MILLISECONDS` ... ouch! 

Please use `Duration` as requested below.

Reply:
Thanks for spotting this. I changed the type of both metrics time parameters to `Duration`.